web/source/dom/domEventHandlers.ts

```diff
@@ -614,8 +619,9 @@ namespace com.keyman.dom {
         }
     
         target.setTextCaret(cp);
-        target.scrollInput();        
-      } else { // Otherwise, if clicked on text in SPAN, set at touch position
+        target.scrollInput();
+        // nextSibling - the scrollbar element.  
+      } else if(tTarg != scroller.nextSibling) { // Otherwise, if clicked on text in SPAN, set at touch position
```

Comment:
What is `scroller.nextSibling` -- why this change?

Reply:
`scroller.nextSibling` - the simulated _scrollbar_.

Not the scrolling `div` used to scroll the element's text contents - the scrollbar element used to control the scrolling position.  Which is therefore _not_ a click within the bounds of the user's text.